static/src/server.ts

```diff
@@ -49,7 +49,19 @@ import { NamedTypedPlace, StatVarSpec } from "../js/shared/types";
 import { TileConfig } from "../js/types/subject_page_proto_types";
 
 const app = express();
-const port = process.env.PORT || 3030;
+const DEFAULT_CONFIG = {
+  port: 8080,
+  apiRoot: "http://mci-website-mcs-svc-ta0n5jd94jdh7kdn:8080",
```

Comment:
can you check how to get a stable server name? This will likely change with new deployment, and will be different across projects.

Reply:
the service name is derived from the MultiClusterService config & can't be controlled by the config .. but updated the deploy script to figure out the service name and then pass into the helm chart as an environment variable for the nodejs server container